server/src-lib/Hasura/GraphQL/Execute/LiveQuery/Poll.hs

```diff
@@ -64,7 +65,13 @@ data Subscriber
   , _sOnChangeCallback :: !OnChange
   }
 
-type OnChange = GQResponse -> IO ()
+-- live query onChange metadata, used for adding more extra analytics data
+data LiveQueryMetadata
+  = LiveQueryMetadata
+  { _lqmExecutionTime :: !(Maybe Double)
```

Comment:
It would be nice to use a more precise type here until we have to convert it to a string—consider making this a `DiffTime`?